model/LoopDistribution/src/inference.py

```diff
@@ -17,13 +17,12 @@
 import traceback
 import sys
 
+import time
```

Comment:
Are we using time?

Reply:
Yes because we are using sleep before reset_pipes. 

Comment added as FIXME.